server/routes/api/chart.py

```diff
@@ -116,12 +117,17 @@ def config(dcid):
     """
     Get chart config for a given place.
     """
+    chart_config = current_app.config['CHART_CONFIG']
+    if os.environ.get('FLASK_ENV') == 'development':
```

Comment:
to understand, in this way, when there is local config change, the server can pick it up without restart?

Reply:
yes, that helps with config changes. though the cache is still in effect on the results, i haven't found a good way to selectively turn that off beyond commenting it out :)